lib/utils.pm

```diff
@@ -1348,4 +1348,15 @@ sub file_content_replace {
     script_run("cat $filename") if $debug;
 }
 
+=head2
+  allow to set break point in test code which could be interrupted from within SUT.
```

Comment:
Sets a break point to pause the test execution. The test execution can be continues by creating a file like `/tmp/break0` inside the SUT.